Source/OxyPlot/Utilities/ArrayExtensions.cs

```diff
@@ -9,11 +9,36 @@
 
 namespace OxyPlot
 {
+    using System;
+    using System.Collections.Generic;
+
     /// <summary>
     /// Provides useful extension methods for arrays.
     /// </summary>
     public static class ArrayExtensions
     {
+        /// <summary>
+        /// Finds the maximum value in the sequence, or returns 0 if the sequence is empty.
+        /// </summary>
+        /// <param name="sequence">The sequence.</param>
+        /// <returns>The maximum value of the sequence, or 0 if the sequency is empty.</returns>
+        public static double MaxOrZero(this IEnumerable<double> sequence)
+        {
+            using var e = sequence.GetEnumerator();
```

Comment:
`null` check would be nice on `sequence`

Reply:
Replaced this by `MaxOrDefault` with a user-defined default value.
Also added a corresponding `MinOrDefault` because it would feel weird otherwise.